### PR TITLE
Remove javascript package as it's moved to web-frontend

### PR DESCRIPTION
**Pull request by joekur**
https://github.com/stitchfix/web-frontend/pull/76 moved the log-weasel JS package to the web-frontend monorepo. This PR removes it from this repo.

https://stitchfix.atlassian.net/browse/KFE-1290

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -80,10 +80,6 @@ jobs:
     - run: bundle install
     - run: bundle exec rspec --format RspecJunitFormatter --out /tmp/test-results/rspec.xml
         --format=doc
-    - run:
-        name: Run Additional CI Steps
-        command: if [ -e bin/additional-ci-steps ]; then bin/additional-ci-steps;
-          fi
```
Comment (joekur) [inline .circleci/config.yml:86]: This script was running yarn lint/test

File: README.md [hunk 2]
```diff
@@ -84,15 +48,15 @@ LogWeasel.configure do |config|
...
+`config.key` (default is the Rails `app_name`)
+A string that will be included in your transaction IDs and is particularly useful in an environment where a unit of work may span multiple applications.
+
+`config.disable_delayed_job_tracing` (default is `false`)
+A boolean that disables Log Weasel appending a `log_weasel_id` parameter in the payloads of delayed Resque jobs. The default is `false`.
 
-`config.disable_delayed_job_tracing` (default is `false`)  
-A boolean that disables Log Weasel appending a `log_weasel_id` parameter in  the payloads of delayed Resque jobs. The default is `false`. 
+`config.debug` (default is `false`)
+A boolean that enables some additional debug logging. The default is `false`.
```
Comment (benmvp) [inline README.md:58]: i'm not sure what the real change was here...

**Review by benmvp — APPROVED** [review]
Had one question. Otherwise looks ez pz!

File: README.md (hunk 2 above)
Comment (joekur) [reply README.md:58]: This is all whitespace changes (thanks to prettier).

**Pull request merged**